_capabilities/utdallas-ufl-xvs.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Explainable Video Activity Search Tool"
+excerpt: "Demo of our explainable query-building tool to search activities within a collection of videos."
+tags: 
+- Analytics
+- Computer Vision
+- Visual Question Answering (VQA)
+- Human-Machine Teaming
+- Explanation Framework
+
+submission_details:
+resources: 
+papers: 
+- title: Anchoring Bias Affects Mental Model Formation and User Reliance in Explainable AI Systems
+url: https://doi.org/10.1145/3397481.3450639
+software:
+- title: Explainable Video Activity Search Demo
+url: https://github.com/MahsanNourani/XAI-Video-Explanation/tree/user-study-v2
+- title: Online Video Activity Search
+url: https://indie.cise.ufl.edu/Pineapple/complex.html
+
+
```

Comment:
Again, strongly recommend a license for the contribution. Please also leave the version, size, and license fields blank if unused:

```suggestion
  version:
  size:
  license:
```